README.md

```diff
@@ -1,60 +1,114 @@
 
-On a [page](https://github.com/settings/developers) with your application you will need `Client ID` and `Client Secret`
+![Screenshot](.github/screenshot.png?raw=true)
 
-Copy `.env.tpl` to `.env` and set tokens there.
+## Requirements
 
-### Docker
+### Global dependencies
 
-```bash
-docker build -t srcd/code-annotation .
-docker run --env-file .env --rm -p 8080:8080 srcd/code-annotation
+You should already have [Go installed](https://golang.org/doc/install#install), and properly [configured the $GOPATH](https://github.com/golang/go/wiki/SettingGOPATH)
+
+```
+go version; # prints your go version
+echo $GOPATH; # prints your $GOPATH path
 ```
 
-### Non-docker
+The project must be under the `$GOPATH`, following the Go import conventions, what means you can go to its directory running:
 
-```bash
-go get github.com/src-d/code-annotation/...
-cd $GOPATH/github.com/src-d/code-annotation
-make serve
+```
+cd $GOPATH/src/github.com/src-d/landing
 ```
 
-## Development
-
-Backend:
+You need also [Yarn v1.x.x installed](https://yarnpkg.com/en/docs/install)
 
 ```
-make gorun
+yarn --version; # prints your Yarn version
 ```
 
-Frontend:
+### Github OAuth tokens
+
+1. You need an OAuth application on GitHub. See [how to create OAuth applications on GitHub](https://developer.github.com/apps/building-oauth-apps/creating-an-oauth-app/).
 
-If you want to benifit from frontend hot reloading feature this line in your `.env` file:
+    `Authorization callback URL: http://127.0.0.1:8080/oauth-callback`
 
+2. Copy `.env.tpl` to `.env`.
+
+3. Retrieve the values for your application's Client ID and Client Secret from the [GitHub Developer Settings page](https://github.com/settings/developers) and add them to the end of the corresponding lines in .env.
+
+## Installation
+
+You need to satisfy all [project requirements](#requirements), and then run:
+
+```bash
+$ go get github.com/src-d/code-annotation/...
+$ cd $GOPATH/github.com/src-d/code-annotation
+$ make serve
 ```
-UI_DOMAIN=http://127.0.0.1:3000
+
+It runs everything you need to get the tool working at [http://localhost:8080](http://localhost:8080)
+
+## Importing and Exporting Data
+
+### Import File Pairs for Annotation
+
+The file pairs must be initially provided via an [SQLite](https://sqlite.org/) database. The database **must follow the expected schema**, please [follow this link](./cli/examples/import/example.sql) to see an example.
+
+The `import` command will use those file pairs to create a new [SQLite](https://sqlite.org/) or [PostgreSQL](https://www.postgresql.org/) database that will be used internally by the Annotation Tool. The destination database does not need to be empty, new imported file pairs can be added to previous imports.
+
+_Please note_: if a file pair is identical to an existing one it will not be detected. A new pair entry will be created with the same contents.
```

Comment:
_Note_: duplicate entries are not filtered, so running an import multiple times will result in repeated rows.